test/test_node_handler.py

```diff
@@ -351,7 +351,7 @@ def test_get_root_node_endpoint(mock_db_find_by_id, mock_init_sub_id):
             name="checkout",
             revision=revision_obj,
             parent="61bda8f2eb1a63d2b7152410",
-            status="pending",
+            state="pending",
```

Comment:
Although the state value doesn't make much of a difference in these tests, the node state value should be changed to the default one.
```suggestion
            state="running",
```
This should be applied to all the nodes in tests.

Reply:
Why does it need to be the default value?

Reply:
I think, If we are setting the random value to node state for tests, the default one should be preferred.
I don't see why we should set it to 'pending'.

Reply:
Well I think we should be testing all the values in principle, and I don't see why the default one is preferred to other ones.  The only reason we might test the default one is if we don't set a value we can check the default one is used when the node is created.

Reply:
Makes sense.